src/modules/tesla-android/filesystem/etc/iptables/rules.v4

```diff
@@ -4,6 +4,7 @@
 :FORWARD ACCEPT [187249:169087507]
 :OUTPUT ACCEPT [1739786:8672196179]
 -A FORWARD -i eth0 -m state --state RELATED,ESTABLISHED -j ACCEPT
+-A FORWARD -i eth1 -m state --state RELATED,ESTABLISHED -j ACCEPT
```

Comment:
Given that one on my LTE sticks exposes itself on eth0 I doubt that this change is required. Could you check if your feature would work without changes to iptables?